src/test/kotlin/org/jetbrains/intellij/tasks/RunPluginVerifierTaskSpec.kt

```diff
@@ -135,6 +139,98 @@ class RunPluginVerifierTaskSpec : IntelliJPluginSpecBase() {
         }
     }
 
+    @Test
+    fun `set verification reports output formats`() {
+        writeJavaFile()
+        writePluginXmlFile()
+        buildFile.groovy(
+            """
+            version = "1.0.0"
+            
+            runPluginVerifier {
+                verificationReportsFormats = [ "markdown", "plain" ]
+                verificationReportsDir = "${'$'}{project.buildDir}/foo"
+                ideVersions = ["IC-2020.2.3"]
+                verifierPath = "/tmp/verifier-cli.jar"
```

Comment:
Why `/tmp/`?

Reply:
Will be removed when the PV is properly released.